translation_v2.json

```diff
@@ -5,7 +5,7 @@
       "collectionName": "cds-feature-attachments",
       "folders": [
         {
-          "startingFolderPath": "cds-feature-attachments/src/main/resources/cds/**/i18n/**",
+          "startingFolderPath": "cds-feature-attachments/src/main/resources/cds/**/_i18n/**",
```

Comment:
First `**` might not be needed, unless we want to deliver more than one model from this plugin.

Reply:
yes, I know, but let's keep it.